bundlewatch.config.json

```diff
@@ -0,0 +1,189 @@
+{
+  "files": [
+    {
+      "path": "./packages/components/pie-assistive-text/dist/index.js",
+      "maxSize": "2.75 KB"
```

Comment:
Super NIT – I think this should be more like 1.3Kb looking at the report?

Reply:
pushed